### PR TITLE
Dojo 6 blog announcement

### PR DESCRIPTION
**Pull request by dylans**
## Description

Dojo 6 blog post (has a few remaining todos)

### Code

This PR touches:

- [x] Content
- [ ] Content Pipeline
- [ ] Frontend
- [ ] Infrastructure

### Tests

- [ ] Tests are included?

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to https://dojoio-ctkdmto4l.now.sh

**Comment by tomdye** [comment]
**Addition re. widgets**

Version 6 of `@dojo/widgets` builds upon it's input widgets by adding `helperText` and a consistent approach to validation. 

`helperText` displays a message below the input it's applied to in order to provide extra context. It also doubles as the location of the error message when used with the `valid` property.

The change from `invalid` to `valid` and `onValidate` provides a mechanism to reactively validate and display error messages beneath a widget. In the case of `text-input`, this is further supplemented by native browser validation meaning that properties such as `required`, `pattern`, `max`, `min` etc will trigger an `onValidate` callback with the result of native validation.

We have added a `Snackbar` and `Card` widget to our library as well as adding `Raised` and `Outline` variations of our `Button`. This is the start of a larger piece of work to support a `material-design` theme in the future as well as a selection of the components it offers.

Finally, we have started writing our new widgets in TSX. We may convert existing widgets over to use TSX in future if we make any extensive changes to them.

File: content/blog/en/version-6-dojo.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+The `.dojorc` configuration for bundle has been enhanced to support globs. This especially useful for scenarios such as internationalization meaning that you don't have to define all the language bundles, instead simply define a matching pattern for each of the locales.
+
+## Update to TypeScript
+
+The minimum required TypeScript version has been updated to 3.4. Updating the core framework to use a recent version enables us to leverage the latest features that underpin the function-based widget and middleware typings.
+
+## Dojo Widgets
+
+// TODO talk about the enhancements made for Dojo 6
```
Comment (dylans) [inline content/blog/en/version-6-dojo.md]:
```suggestion
Version 6 of `@dojo/widgets` builds upon its input widgets by adding `helperText` and a consistent approach to validation.

`helperText` displays a message below the input to which it gets applied to provide extra context. The `helperText` also doubles as the location of the error message when used with the `valid` property.

The change from `invalid` to `valid` and `onValidate` provides a mechanism to reactively validate and display error messages beneath a widget. In the case of `text-input`, this is further supplemented by native browser validation meaning that properties such as `required`, `pattern`, `max`, `min`, etc. will trigger an `onValidate` callback with the result of native validation.

We have added `Snackbar` and `Card` widgets to the Dojo widget library as well as adding `Raised` and `Outline` variations of the `Button` widget. This is the start of a larger initiative to support a material design theme in the future as well as a selection of the components material design offers.

Finally, we have started writing our new widgets using TSX syntax. We may convert existing widgets to use TSX in the future if we make any extensive changes to them.
```

**Comment by dojo-site** [comment]
This pull request has been deployed to https://dojoio-kjbi3lpmc.now.sh

**Comment by agubler** [comment]
closing as I'm making changes to the blog and will re-raise.

**Comment by agubler** [comment]
Raised here https://github.com/dojo/site/pull/151

**Pull request closed without merging**